api/v1/placementbinding_types.go

```diff
@@ -35,6 +36,8 @@ type PlacementBinding struct {
 	// +kubebuilder:validation:Required
 	PlacementRef Subject `json:"placementRef"`
```

Comment:
Could you please create a separate struct type for `PlacementRef` since your code change assumes that any subject in the `Subjects` slice will be a Policy, but the enum also allows for `PlacementRule` and `Placement`?

Reply:
done